pom.xml

```diff
@@ -0,0 +1,703 @@
+        <artifactId>junit-jupiter-engine</artifactId>
+        <version>${version.org.junit.jupiter}</version>
+        <scope>test</scope>
+      </dependency>
+
+      <!--
+      Transitive Dependency Management
+
+      In this section we address build issues including security vulnerabilities
+      in transitive dependencies we don't explicitly declare above or in the submodules
+      Versioning and scope declared here will override anything inherited through
+      transitivity, so use with care. Also note: these are in place for a
+      point in time. As we maintain this software, the manual forced resolution we do
+      here may:
+
+      1) No longer be necessary (if we have removed a dependency path leading to dep)
+      2) Break an upgrade (if we upgrade a dependency and this forces a lower version
+         of a transitive dependency it brings in)
+
+      So we need to exercise care here, and, when upgrading our deps, check to see if
+      these overrides of transitive dependency versions and scope aren't breaking things.
+
+      When adding an entry here, please reference the issue which explains why we
+      needed to do this; it will help future maintainers understand if the force
+      is still valid, should be removed, or handled in another way.
+
+      When in doubt, ask! :)
+      -->
+      <!--
+
+      FasterXML Jackson -
+
+      Needed for libraries to play nicely together and avoid NoClassDefFoundError
+      -->
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-annotations</artifactId>
+        <version>${version.com.fasterxml.jackson}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-core</artifactId>
+        <version>${version.com.fasterxml.jackson}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-databind</artifactId>
+        <version>${version.com.fasterxml.jackson}</version>
+      </dependency>
+      <!-- Addresss https://github.com/TBD54566975/web5-kt/issues/243 -->
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>${version.com.google.guava}</version>
+      </dependency>
+      <!-- Addresss https://github.com/TBD54566975/web5-kt/issues/242 -->
+      <dependency>
+        <groupId>com.google.protobuf</groupId>
+        <artifactId>protobuf-javalite</artifactId>
+        <version>${version.com.google.protobuf}</version>
+      </dependency>
+      <!-- Addresses https://github.com/TBD54566975/web5-kt/issues/244 -->
+      <dependency>
+        <groupId>com.squareup.okio</groupId>
+        <artifactId>okio</artifactId>
+        <version>${version.com.squareup.okio}</version>
+      </dependency>
+
+    </dependencies>
+  </dependencyManagement>
+
+  <!-- Dependencies -->
```

Comment:
```suggestion
  <!-- Test Dependencies -->
```
?

Reply:
Reviewed w/ @jiyoontbd - these are the common deps for all submodules, both test and runtime